docs/topics/development/dependencies.md

```diff
@@ -0,0 +1,62 @@
+# Project Dependencies
+
+This document describes how to add/upgrade dependencies in the project.
+We use pip to manage dependencies and hashin to lock versions. We use npm to manage frontend dependencies.
+
+## Python
+
+### Adding Python Dependencies
+
+We have 2 requirements files for python dependencies:
+
+- prod.txt
+- dev.txt
+
+Prod dependencies are used by our django app in runtime.
+They are strictly required to be installed in the production environment.
+
+Dev dependencies are used by our django app in development or by tools we use for linting, testing, etc.
+
+To ensure a standard and sane process for installing packages,
+use the `make install` command to install a new package.
+This will add your specified version to one of our requirements/(dev|prod).txt files and
+install the package in your local container.
+
+```bash
+make install [package]==[version] --dev --prod
+```
+
+Note: This script is strict. You must specify a version, and you must specify either dev or prod.
+
+### Upgrading Python Dependencies
+
+TBD
```

Comment:
missing content?

Reply:
Yep, how do we normally handle upgrades? is it exclusively dependabot? Do we have a process for managing dependabots typically? IDK what to put but knew there should be something, hence the empty section.

Reply:
Yeah, dependabot. But ... well, there are always exceptions

Reply:
IF you write some of the exceptions you can think of in this thread, I can add them to the doc.